--- core/mui/gtk_postbox.rb.org	2012-02-29 11:33:51.421695468 +0900
+++ core/mui/gtk_postbox.rb	2012-03-05 23:24:16.380108622 +0900
@@ -8,6 +8,7 @@
 require 'thread'
 miquire :mui, 'miracle_painter'
 miquire :mui, 'intelligent_textview'
+miquire :mui, 'emacslike_textview'
 
 module Gtk
   class PostBox < Gtk::EventBox
@@ -62,7 +63,7 @@
 
     def widget_post
       return @post if defined?(@post)
-      @post = Gtk::TextView.new
+      @post = Gtk::EmacsLikeTextView.new
       post_set_default_text(@post)
       @post.wrap_mode = Gtk::TextTag::WRAP_CHAR
       @post.border_width = 2
@@ -136,6 +137,7 @@
       if postable?
         return unless before_post
         text = widget_post.buffer.text
+        widget_post.pushGlobalStack
         text += UserConfig[:footer] if add_footer?
         @posting = service.post(:message => text){ |event, msg|
           notice [event, msg].inspect
